sagemaker-experiments/notebook_tracking/pytorch_experiment.ipynb

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    experiment_name = "local-experiment-example-{}".format(datetime.now().strftime("%d-%m-%Y-%H-%M-%S"))
</pre><p>FYI: there's an helper function to do the same in SM PySDK. This is just a FYI, and we can keep the existing way of generating the experiment name</p><pre class="ql-syntax" spellcheck="false">from sagemaker.utils import unique_name_from_base
experiment_name = unique_name_from_base("local-experiment-example")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3703/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>